flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -193,109 +192,106 @@ private static void assertEquals(
       return;
     }
 
-    Assert.assertTrue(
-        "expected and actual should be both null or not null", expected != null && actual != null);
+    assertThat(expected).isNotNull();
+    assertThat(actual).isNotNull();
 
```

Comment:
from the previous assertion I can read:
```"expected and actual should be both null or not null```
however, this block
```java
assertThat(expected).isNotNull();
    assertThat(actual).isNotNull();
```
is only asserting that both of them are not null

Reply:
I think the description is just misleading, because the check actually makes sure that both are not null